src/Microsoft.VisualStudio.ProjectSystem.IntegrationTests/OpenProjectTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.Test.Apex.VisualStudio;
+using Microsoft.Test.Apex.VisualStudio.Solution;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.VisualStudio.ProjectSystem.IntegrationTests
+{
+    [TestClass]
+    public class CreateProjectTests : VisualStudioHostTest
+    {
+        private static string _hiveName;
+
+        [ClassInitialize()]
+        public static void ClassInitialize(TestContext context)
+        {
+            _hiveName = context.Properties["VsRootSuffix"].ToString();
+        }
+
+        protected VisualStudioHostConfiguration DefaultHostConfiguration
+        {
+            get
+            {
+                var visualStudioHostConfiguration = new VisualStudioHostConfiguration()
+                {
+                    CommandLineArguments = $"/rootSuffix:{_hiveName}",
```

Comment:
Visual studio doesn't allow the `:` here.

Reply:
yep, always forget that this is different... fixed